grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcJsonMarshaller.java

```diff
@@ -39,19 +39,46 @@
 
     /**
      * Returns a newly-created {@link GrpcJsonMarshaller} which serializes and deserializes a {@link Message}
-     * served by the {@linkplain ServiceDescriptor service}.
+     * served by the {@linkplain ServiceDescriptor service}. This implementation internally uses
+     * <a href="https://github.com/curioswitch/protobuf-jackson">protobuf-jackson</a>
+     * to serialize and deserialize messages.
      */
     static GrpcJsonMarshaller of(ServiceDescriptor serviceDescriptor) {
         return builder().build(serviceDescriptor);
     }
 
     /**
-     * Returns a new {@link GrpcJsonMarshallerBuilder}.
+     * Returns a new {@link GrpcJsonMarshallerBuilder}. This implementation internally uses
+     * <a href="https://github.com/curioswitch/protobuf-jackson">protobuf-jackson</a>
+     * to serialize and deserialize messages.
      */
     static GrpcJsonMarshallerBuilder builder() {
         return new GrpcJsonMarshallerBuilder();
     }
 
+    /**
+     * Returns a newly-created {@link GrpcJsonMarshaller} which serializes and deserializes a {@link Message}.
+     * This implementation internally uses
+     * <a href="https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/util/JsonFormat">JsonFormat</a>
+     * and {@code Gson} to serialize and deserialize messages.
+     * Note that using {@link GrpcJsonMarshaller#of(ServiceDescriptor)} yields better performance in general
+     * and should be preferred.
```

Comment:
Shall we discourage using it unless absolutely necessary? e.g.

```suggestion
     * Note: <a ...>JsonFormat</a> has a known performance issue. Use this method only when
     * 1) you're using Protobuf 2 or 2) you have an issue with <a ...>protobuf-jackson</a>.
```